src/style.css

```diff
@@ -0,0 +1,105 @@
+* {
+  padding: 0;
```

Comment:
do not reset padding for all selectors. just where it is necessary

Reply:
Thank you
I will remove it now.